api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -67,4 +67,19 @@ message BandwidthLimit {
   // Runtime flag that controls whether the filter is enabled or not. If not specified, defaults
   // to enabled.
   config.core.v3.RuntimeFeatureFlag runtime_enabled = 5;
+
+  // Optional Flag to enable response trailers.
+  //
+  // .. note::
+  //   If set true, the response trailer "bandwidth-request-delay-ms" and "bandwidth-response-delay-ms"
+  //   will be added.
+  //   bandwidth-request-delay-ms: delay time in milliseconds it took for the request stream transfer.
```

Comment:
Make these RST lists, I suggest looking at docs rendering preview for the API docs for this filter.